### PR TITLE
Standardised Contained Sprites : Now Atomic

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title. Also might as well get the Bridge Crew uniforms squared away. 
More *atomic* than the last PR. 
Focused on getting rid of the *egregeous* other standards rather than sorting everything at once. 
That comes later. Also just changes the file path of machinery. That should be it.

Addendum : Hey, see that random mapping change? That's a fix. Turns out that sprite was invisible...because people still hard-code in things in maps. I'm going to kill.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by alsoandanswer** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

File: code/modules/clothing/under/accessories/holster.dm [hunk 1]
```diff
@@ -182,8 +184,6 @@
 /obj/item/clothing/accessory/holster/utility
 	name = "utility holster"
 	desc = "A utility holster."
-	icon = 'icons/contained_items/clothing/accessories/utility_holsters.dmi'
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/accessories/holster.dm:185]: keep this as-is. the utility holsters have minor tweaks to the sprite.

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/obj/item/gun/energy/gun
+/obj/item/gun/energy/gun // what idiot made this path
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/nuclear.dm:1]:
```suggestion
/obj/item/gun/energy/gun
```
leftover comment.

**Review by SleepyGemmy — DISMISSED** [review]

File: code/modules/clothing/under/accessories/accessory.dm [hunk 3]
```diff
@@ -660,8 +660,7 @@
 	desc_extended = "A cloak given to senior level doctors and researchers for Zeng-Hu who has \
 	in the past been given the privilege of working within or in collaboration with the Nralakk Federation\
 	 as a show of goodwill between the corporation and federation."
-	icon = 'icons/obj/contained_items/accessories/ZH_cape.dmi'
-	icon_override = 'icons/obj/contained_items/accessories/ZH_cape.dmi'
+	icon = 'icons/obj/item/clothing/accessory/ZH_cape.dmi'
```
Comment (alsoandanswer) [inline code/modules/clothing/under/accessories/accessory.dm:663]:
```suggestion
	icon = 'icons/obj/item/clothing/accessory/zh_cape.dmi'
```

**Comment by SleepyGemmy** [comment]
`icons/obj/clothing/ties.dmi` and `icons/mob/ties.dmi` have the bowtie sprite due to merge master, but they shouldn't.

File: code/modules/clothing/under/accessories/holster.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/modules/clothing/under/accessories/holster.dm:185]: correct me if i'm wrong, but there is no difference between the brown holsters and the utility holsters whatsoever.

Comment (SleepyGemmy) [reply code/modules/clothing/under/accessories/holster.dm:185]: there is.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
restore utility holster sprites, at least the hip one.
`icons/obj/clothing/ties.dmi` and `icons/mob/ties.dmi` have the bowtie sprite due to merge master, but they shouldn't.

**Pull request merged**